src/pipe.toit

```diff
@@ -383,6 +383,7 @@ Forks a program, and returns the exit status.
 A return value of zero indicates the program ran without errors.
 Uses the /bin/sh shell to parse the command, which is a single string.
 Arguments are split by the shell at unescaped whitespace.
+  On Windows we just split at spaces, since the shell is not available.
```

Comment:
Wouldn't it be nicer to try to be a bit smarter? Like `system "\"foo bar\""` should probably not be split.

Reply:
I think it's not worth it.  Added comment to direct users to run_program for non-trivial cases.